python/scripts/prepare_dolmino.py

```
@@ -0,0 +1,378 @@
+
+No quality score filter — Dolmino-Mix is already curated upstream.
+No tokenizer training — reuses data/fineweb_edu/tokenizer.json.
+
+Default --min_text_len=2048 (~512 tokens) ensures every document in the stream
+can span at least one full L3 CMS period. Shorter documents are discarded.
+
+Usage:
+    python scripts/prepare_dolmino.py
+    python scripts/prepare_dolmino.py --target_tokens 1_000_000_000
+    python scripts/prepare_dolmino.py --ingredient ingredient2 --output data/dolmino_i2
+    python scripts/prepare_dolmino.py --min_text_len 4096   # stricter: force 2× L3
```

Ruff (0.15.2)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)